Makefile

```diff
@@ -8,9 +8,9 @@ release: clean dist upload
 requirements:
 	sudo pip install -r dev-requirements.txt --upgrade
 format:
-	./format.sh
+	python3 ./format.py
 checkformat:
-	./checkformat.sh
+	python3 ./check_format.py
```

Comment:
Please change `python3` into `python`. It should be matter of virtualenv or system config. Once python3 is set properly, it can be invoked via `python` command.